ui/vite.config.js

```diff
@@ -29,6 +30,18 @@ export default defineConfig({
                 }
             ]
         }),
+        downloadVsix({
+            targets: [
+                {
+                    vsixUrl: 'https://github.com/kestra-io/vscode-kestra/releases/download/v0.1.7/ms-python.vscode-pylance-2023.11.12.vsix',
```

Comment:
Isnt there a latest for that ?

Reply:
It's never a good idea to fetch latest as it's not an opt-in update in our case which could lead to breaking all our users.

However I admit that the "upload vsix to github release" is not ideal but I couldn't find any other way around as VSCode marketplace seems to limit programmatical fetches (rate limiter + cookie checker), see comment above.